x/genutil/v2/types.go

```diff
@@ -20,4 +22,6 @@ type ExportedApp struct {
 	AppState json.RawMessage
 	// Height is the app's latest block height.
 	Height int64
+	// Validators is the exported validator set.
+	Validators []sdk.GenesisValidator
```

Comment:
Can we populate this field in simapp v2 export

Reply:
done